net-analyzer/authforce/authforce-0.9.9-r2.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="An HTTP authentication brute forcer"
+HOMEPAGE="http://www.divineinvasion.net/authforce/"
+SRC_URI="http://www.divineinvasion.net/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="curl nls"
+
+DEPEND=${RDEPEND}
```

Comment:
either `DEPEND` comes first and defines all the deps, which `RDEPEND` then inherits, or `DEPEND` inherits properly *after* `RDEPEND` was defined. ;)